.gitignore

```diff
@@ -8,3 +8,8 @@
 .pytest_cache
 __pycache__
 .hypothesis
+
+dist/
+.DS_Store
+
+data.sqlite
```

Comment:
Don't think this is needed?

Reply:
you wanna commit these?

Reply:
> you wanna commit these?

There's nothing here that generates this file?

Reply:
Not super bothersome, just think it has to do with chaosnet, not this repo